### ISSUE TITLE
Easy villagers 

### ISSUE THREAD

**Issue by tyler489** [EnigmaticaModpacks/Enigmatica6#4574]
### URL

https://www.curseforge.com/minecraft/mc-mods/easy-villagers

### Why would you like the mod added?

I run a small server of about 10 people on at any time, on average, 40 villagers are loaded at any given time and take up anywhere from 10-85ms due to their attempt at pathfinding. Easy villagers allows you to pickup and store villagers inside the  trader block, so they do not have a pathfinding qi anymore, I would love a way to keep our villagers on the server without having Massive Tps drops from villagers.

**Comment by tyler489** [comment]
I saw issue #640, maybe leave only the trader and breeder blocks enabled and put them as a midgame item, say mekanism diamond alloy or something simular?

**Comment by NillerMedDild** [comment]
I'm sympathetic to the issue, but I still don't want to add the mod. Perhaps we can find a way to allow players to disable villager AI once they have the traded they want.

**Comment by TemperedFool** [comment]
Would Create Seats or IF Stasis Chamber serve that function?

**Comment by Discomanco** [comment]
That could maybe cause issues with interacting with workstations.  
A vanilla method is to put magma blocks under their workstations (assuming 1x1 holding chambers), which would block their pathfinding further than 1 block

**Comment by TemperedFool** [comment]
Yeah but once their profession is locked in, they don't need their workstations right?

**Comment by Discomanco** [comment]
They need their workstations to restock

**Comment by TemperedFool** [comment]
This is why I don't run a villager farm 🤣

**Comment by tyler489** [comment]
> I'm sympathetic to the issue, but I still don't want to add the mod.
> 
> Perhaps we can find a way to allow players to disable villager AI, what do you think?

I tried using commands to disable Ai and that also removes their ability to restock unfortunately.

**Comment by MuteTiefling** [comment]
> Would Create Seats or IF Stasis Chamber serve that function?

I can confirm that they restock fine while seated in a Create seat, and this disables their pathfinding AI as well.

**Comment by theboo** [comment]
Personally recommend the server getting together and creating a public trading hall, and have them set on create seats which disabled their pathfinding, but they are still able to restock with adjacent workstations

**Comment by MuteTiefling** [comment]
Absolutely. Servers are the best place to share resources like that... Same goes for things like Mek Fission/fusion. Why do you need 1 per player? 

It's a perfect excuse to go have fun building a big building together that you can all enjoy.

**Comment by TheDragonicSlayer** [comment]
If I may, there are also servers like the ones I play with my friends where the whole server is about PVP, raiding, etc.

**Comment by jlfree12** [comment]
If you’re going to go the route of recommending the create seats you should add it as a loading screen tip.

**Comment by tyler489** [comment]
we setup a trading hall, I was unaware create seats would allow them to still restock, unfortunately they can't breed while seated, so still going to have them moving while trying to get more

**Comment by TheDragonicSlayer** [comment]
I personally think create seats should be disable because they allow you to cheese bosses

**Comment by jlfree12** [comment]
can’t you also cheese bosses with minecarts/boats etc.

**Comment by tyler489** [comment]
> 

Yep, thats all I wanted, but oh well.

**Comment by MuteTiefling** [comment]
You're always free to add it yourself.
